src/utils/index.js

```diff
@@ -1,2 +1,14 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const emailVerifyToken = (id) => new Promise((resolve) => {
+  const token = jwt.sign({ id }, process.env.SECRET_KEY_EMAIL_VERIFY_TOKEN, { expiresIn: '24h' });
+  if (token) resolve(token);
+});
+
 const arrayTest = (array, index) => array.indexOf(index);
-export default arrayTest;
+
+
+export { emailVerifyToken, arrayTest };
```

Comment:
Please move this ticket back to `peer review` while Implement feedback.

Reply:
Okay